recruitment/views.py

```diff
@@ -246,6 +252,40 @@ def json(self):
                     [arguments.get(pk=int(answer.answer)).value for answer in custom_field.customfieldanswer_set.all()]
                 )
                 custom_field_counts.append(custom_field_count)
+
+
+
+                # Also, for each argument we want to plot bar graphs so we can see the number of english or swedish applicants per date
+                argument_per_date_counts = []
+                y_limit = 0
+                for argument in arguments:
+                    date_dictionary = dict(
+                        [(date_filter(application.submission_date, "d M"), application.submission_date) for application in
+                         application_list])
+
+                    argument_per_date_count = ValueCounter(
+                        argument.value,
+                        True,
+                        ['bar'],
+                        [date_filter(application.submission_date, "d M") for application in application_list if CustomFieldAnswer.objects.filter(user=application.user, answer=str(argument.pk)).exists()],
+                        lambda x: date_dictionary[x]
+                    )
+
+                    for date in daterange(recruitment_period.start_date, min(timezone.now(), recruitment_period.end_date)):
+                        date_string = date_filter(date, "d M")
+                        if not date_string in argument_per_date_count.data:
```

Comment:
Please change to `if date_string not in argument_per_date_count.data:` as this complies with PEP8.